app/policies/user_creation_policy.rb

```diff
@@ -1,35 +1,32 @@
 class UserCreationPolicy < ApplicationPolicy
-  # Defines the roles that allow admins to modify user creations.
   # User creations are Bookmarks, ExternalWorks, Series, Works.
+
+  # Roles that allow destroying user creations.
   DESTROY_ROLES = %w[superadmin policy_and_abuse].freeze
-  # Support admins need edit permissions due to AO3-4932.
-  EDIT_ROLES = %w[superadmin support policy_and_abuse].freeze
-  HIDE_ROLES = %w[superadmin policy_and_abuse].freeze
-  SPAM_ROLES = %w[superadmin policy_and_abuse].freeze
 
-  def self.can_destroy_creations?(user)
-    self.new(user, nil).can_destroy_creations?
-  end
+  # Roles that allow destroying only Works.
+  #
+  # Include support admins for handling FNOK requests.
```

Comment:
I was like, "Wait, what? FNOK doesn't relate to deleting works," but I think I see where the confusion came from. The issue wants us to allow Support to destroy works not because they're taking over FNOK management, but because they're also taking over management of duplicate works.

However, I do have a little lingering confusion -- why does PAC need to be here? Does `DESTROY_ROLES` not cover destroying works, only other types of creations?

Reply:
I'll update the comment.

`DESTROY_ROLES` covers all types of user creations, so having PAC and super admins in `DESTROY_WORK_ROLES` is to be technically true, but it's not necessary. Should I remove both?

Reply:
I think the relationship between `DESTROY_ROLES` and `DESTROY_WORK_ROLES` is clearer if we just let it be support, yeah. Maybe we should also clarify the comment on `DESTROY_ROLES` to say it's for all types of user creations just to be extra clear.